help/md/mcmcmc.md

```diff
@@ -7,10 +7,12 @@ The Mcmcmc analysis object keeps a model and the associated moves and monitors.
 ## details
 The Mcmcmc analysis object produced by a call to this function keeps copies of the model and the associated moves and monitors. The Mcmcmc analysis object is used to run Markov chain Monte Carlo (Mcmcmc) simulation on the model, using the provided moves, to obtain a sample of the posterior probability distribution. During the analysis, the monitors are responsible for sampling model parameters of interest.
 
-An MCMCMC analysis is initiated using the `mcmcmc.run()` method.  
-The `StoppingRule[]` argument provides a mechanism to automatically terminate a run once a set of rules are met: perhaps once the run has attained convergence, or after a certain amount of time has passed.  The run will be terminated once *all* convergence rules ([`srGelmanRubin()`], [`srGeweke()`], [`srMinESS()`], [`srStationarity()`]) have been fulfilled; or once *any* threshold rules ([`srMaxTime()`], [`srMaxIteration()`]) are met.
```

Comment:
What removing the comments about StoppingRule intentional?

Reply:
Hmm? Nothing of substance was removed, just rephrased in accordance with changes made to `mcmc.md` by PR #505.